api-spec/api.yaml

```diff
@@ -481,12 +481,7 @@ components:
           type: string
           description: Identifier of the payment gateway session associated to the form
         fields:
-          oneOf:
-            - $ref: "#/components/schemas/CardFormFields"
-          discriminator:
-            propertyName: paymentMethod
-            mapping:
-              CARD: "#/components/schemas/CardFormFields"
+            $ref: "#/components/schemas/CardFormFields"
```

Comment:
why delete multiple methods with the same model?

Reply:
Because of code generation issues with the mock. Given that initially we are considering cards only we can update the API later and have only the card variant here